.github/workflows/mac-compatability.yml

```diff
@@ -0,0 +1,29 @@
+# GitHub Actions for genCMPClient
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) Siemens AG, 2021-2024
+
+name: macos-CI
+
+on: push
+
+jobs:
+  cmake:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: cmake
+        run: |
+          cmake .
+          make
+          mkdir build
+          cd build
+          cmake -S .. -B .
+          cmake --build .
+          cmake -DCMAKE_BUILD_TYPE=Release ..
+          make clean build
+          cd ..
+          mkdir build-with-libcmp
+          cd build-with-libcmp
+          USE_LIBCMP=1 cmake -S .. -B .
+          make clean build
+          DESTDIR=tmp make install uninstall
```

Comment:
Please also add 
```
cd ..

make -f Makefile_v1 test_Mock
```

Reply:
Argh, with macOS the HTTP-based tests hand, I believe due to waiting for interactive approval for the server to receive input on a local port.

If we do not quickly find way of fixing this, I suggest replacing
```
make -f Makefile_v1 test_Mock`
```
by just
```
make -f Makefile_v1
./cmpClient --help
```